nhsuk/settings/production.py

```diff
@@ -2,7 +2,54 @@
 
 from .base import *  # noqa
 
-DEBUG = False
+import os
+
+ALLOWED_HOSTS = ['*']
```

Comment:
Let's make a note that we need to change this to not using `*` if we don't know yet what should go there.